File: DojranSteelSPA/src/app/_models/meshProduct.ts
export interface MeshProduct {
    id: number;
    sapCode: number;
    description: string;
    materialGroupNumber: number;
    productionLine: string;
    theoreticalWeight: number;
    baseUnitOfMeasure: string;
    width: number;
    lenght: number;
    lineWireDistance: number;
    crossWireDistance: number;
    rawLineWireSize: number;
    rawCrossWireSize: number;
    lineWireSize: number;
    crossWireSize: number;
    piecesInBundle: number;
    numberOfLineWires: number;
    numberOfCrossWires: number;
    weightOfLineWires: number;
    weightOfCrossWires: number;
    maxWeldsPerMinute: number;
    budgetedEficiency: number;
    budgetedWeldsPerMinute: number;
    bundleWeight: number;
}

File: DojranSteel.API/Models/MeshProduct.cs
namespace DojranSteel.API.Models
{
    public class MeshProduct: Product
    {
        public decimal Width { get; set; }
        public decimal Lenght { get; set; }
        public decimal LineWireDistance { get; set; }
        public decimal CrossWireDistance { get; set; }
        public decimal RawLineWireSize { get; set; }
        public decimal RawCrossWireSize { get; set; }
        public decimal LineWireSize { get; set; }
        public decimal CrossWireSize { get; set; }
        public int PiecesInBundle { get; set; }
        public int NumberOfLineWires { get; set; } 
        public int NumberOfCrossWires { get; set; }
        public decimal WeightOfLineWires { get; set; }
        public decimal WeightOfCrossWires { get; set; }
        public decimal MaxWeldsPerMinute { get; set; }
        public decimal BudgetedEficiency { get; set; }
        public decimal BudgetedWeldsPerMinute { get; set; }
        public decimal BundleWeight { get; set; }
    }
}